src/pages/policy/PolicyRightSidebar.jsx

```diff
@@ -41,29 +40,26 @@ function PolicyNamer(props) {
 }
 
 function SinglePolicyChange(props) {
-  const { startDateStr, endDateStr, parameterMetadata, value } = props;
+  const { startDateStr, endDateStr, parameterMetadata, value, paramLabel } = props;
+  const oldVal = getParameterAtInstant(parameterMetadata, startDateStr);
+  const oldValStr = formatVariableValue(parameterMetadata, oldVal);
   const newValueStr = formatVariableValue(parameterMetadata, value);
-  const startDate = moment(startDateStr).format("LL");
-  const endDate = moment(endDateStr).format("LL");
+
   return (
-    <div style={{ display: "flex", alignItems: "center" }}>
-      <div
-        style={{
-          width: 200,
-          textAlign: "right",
-          marginRight: 10,
-          flex: 3,
-          paddingRight: 5,
-          borderRightWidth: 2,
-          borderRightStyle: "solid",
-          borderRightColor: style.colors.DARK_GRAY,
-        }}
-      >
-        <div>from {startDate}</div>
-        <div>until {endDate}</div>
+    <div 
+      style={{ 
+        display: "flex", 
+        flexDirection: "column",
+        alignItems: "flex-start",
+        padding: "0 10%" }}
+    >
+      <div style={{ textAlign: "left" }}>
+          {paramLabel} from{' '}
+          <span className="text-secondary fw-bold">{oldValStr}</span> to{' '} 
+          <span className="text-primary fw-bold">{newValueStr}</span>
```

Comment:
Could we avoid colouring the text blue?

Reply:
Ah sorry ignore this - just remembered we did decide to make this blue. Though could we use the same blue as the rest of the site? It's in `style.colors.BLUE`.